ddtrace/settings/dynamic_instrumentation.py

```diff
@@ -131,3 +132,4 @@ class DynamicInstrumentationConfig(En):
 
 
 config = DynamicInstrumentationConfig()
+_report_telemetry(config)
```

Comment:
Yup. This would be ideal. The main issue here is that most subclasses of `En` do not initialize the super class. So we will still need to make a code change in 10 different places 😞